deploy/examples/nfs-test.yaml

```diff
@@ -4,14 +4,24 @@ metadata:
   name: my-nfs
   namespace: rook-ceph # namespace:cluster
 spec:
+  # For Ceph v16, use the block here.
   # rados settings aren't necessary in Ceph Versions equal to or greater than Pacific 16.2.7
   rados:
     poolConfig:
       failureDomain: osd
       replicated:
         size: 1
-    # RADOS namespace where NFS client recovery data is stored in the pool.
-    namespace: nfs-ns
+
+  # For Ceph v15, use the block here.
```

Comment:
Do we care for the `-test.taml`? The cluster-test.yaml points to v16.2 already so I don't think we need to repeat this.

Reply:
Yeah, I can remove the changes in `nfs-test.yaml`